llvm/lib/Support/Unix/Threading.inc

```diff
@@ -432,6 +436,8 @@ static int computeHostNumPhysicalCores() {
       static_cast<uintptr_t>(reinterpret_cast<unsigned int &>(CVT[CVTCSD])));
   return reinterpret_cast<int &>(CSD[CSD_NUMBER_ONLINE_STANDARD_CPS]);
 }
+#elif defined(_AIX)
+static int computeHostNumPhysicalCores() { return _system_configuration.ncpus; }
```

Comment:
```suggestion
static int computeHostNumPhysicalCores() { return sysconf(_SC_NPROCESSORS_ONLN); }
```

Reply:
If we want to do this version then we can opt into the `elif defined(__linux__) && defined(__s390x__)` block above which has this implementation (is it really better to make the syscall though rather than reading the structure?)

Reply:
It will be good to common up with `__linux__` and `__s390x__`.  The structure has the number of CPUs configured but `_SC_NPROCESSORS_ONLN` gives the number of processors currently online and available to handle tasks.